lib/commands/publish-draft.coffee

```diff
@@ -21,6 +21,9 @@ class PublishDraft
 
     @confirmPublish =>
       try
+        postassetfolder = path.join(path.dirname(@draftPath), utils.getTitleSlug(@draftPath))
+        fs.moveSync(postassetfolder,
+                    path.join(path.dirname(@postPath), @_getPostTitle())) if fs.existsSync(postassetfolder)
```

Comment:
I am uncertain about `publish-draft` should move the asset folder. If so, I believe create draft/post need to create the asset folder as well.

Hexo users will probably rely on hexo command provided?